.pre-commit-config.yaml

```diff
@@ -13,59 +12,60 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.6.0
     hooks:
-      - id: trailing-whitespace # Removes trailing whitespaces from lines
-      - id: end-of-file-fixer # Ensures files end with a newline
-      - id: check-shebang-scripts-are-executable # Checks that scripts with shebangs are executable.
-      - id: check-case-conflict # Check conflict on a case-insensitive filesystem (MacOS HFS+/Windows FAT).
-      - id: mixed-line-ending # Detects mixed line endings (CRLF/LF)
-        args: ['-f=lf'] # Forces files to use LF line endings
-
-  - repo: https://github.com/pylint-dev/pylint
-    rev: v3.2.6
-    hooks:
-      - id: pylint # Runs pylint on Python code
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-shebang-scripts-are-executable
+      - id: check-case-conflict
+      - id: mixed-line-ending
+        args: ['-f=lf']
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.6.3
     hooks:
-      - id: ruff # Linter
+      - id: ruff
         args: [--fix, --exit-non-zero-on-fix]
-      - id: ruff-format # Formatter (replaces Black)
+      - id: ruff-format
 
   - repo: https://github.com/pre-commit/mirrors-mypy
     rev: v1.11.2
     hooks:
-      - id: mypy # Runs mypy for Python type checking
+      - id: mypy
+
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.17.0
+    hooks:
+      - id: pyupgrade
+        args: [--py39-plus]
```

Comment:
could be done by ruff `UP`

Reply:
@hfudev Thank you so much for checking this.

> Would the mypy check fail if the cast is removed?

Yes - this is altering of Commitizen default settings and they return type `Any`. In our mypy rules we prohibit this return type, which causes fail. Casting is here just to satisfy type checker.

Reply:
> could be done by ruff UP

This is great, thank you. Updated.